chore(weekly): add blog post for 26/2024 (#901)
packit/packit.dev: weekly/2024/week-26.md

---
title: Week 26 in Packit
date: 2024-07-01
authors: mfocko
tags:
  - 2024-June
  - 2024
  - June
---

## Week 26 (June 25th – July 1st)

> - Enable HTTP/2 for Packit service ([packit-service#2450](https://github.com/packit/packit-service/pull/2450))
> - It is now possible to disable reporting issues about failure of `propose_downstream` by setting `notifications.failure_issue.create` in configuration to `false`. ([packit-service#2451](https://github.com/packit/packit-service/pull/2451))

Reviewer: ```suggestion
- Enabled HTTP/2 for Packit service ([packit-service#2450](https://github.com/packit/packit-service/pull/2450))
- It is now possible to disable reporting of `propose_downstream` failures by setting `notifications.failure_issue.create` in the configuration to `false`. ([packit-service#2451](https://github.com/packit/packit-service/pull/2451))
```
